epigraphhub/data/sinan_fetch.py

```diff
@@ -0,0 +1,143 @@
+from typing import Tuple
+
+import os
+import time
+from pathlib import Path
+
+import pandas as pd
+import psycopg2
+import psycopg2.extras as extras
+import pyarrow.parquet as pq
+from dotenv import load_dotenv
+from pandas.io.sql import get_schema
+from pysus.online_data.SINAN import download
+from sqlalchemy import create_engine
+
+load_dotenv()
+env_path = Path(".") / ".env"
+load_dotenv(dotenv_path=env_path)
+
+
+db_config = {
+    "database": os.getenv("POSTGRES_EPIGRAPH_DB"),
+    "user": os.getenv("POSTGRES_USER"),
+    "password": os.getenv("POSTGRES_PASSWORD"),
+    "host": os.getenv("POSTGRES_HOST"),
+    "port": os.getenv("POSTGRES_PORT"),
+}
+
+PSQL_URI = "postgresql://{}:{}@{}:{}/{}".format(
+    db_config.get("user"),
+    db_config.get("password"),
+    db_config.get("host"),
+    db_config.get("port"),
+    db_config.get("database"),
+)
+
+engine = create_engine(PSQL_URI)
+conn = psycopg2.connect(**db_config)
```

Comment:
We have the functions that Ivan created in this script to make the connection: https://github.com/thegraphnetwork/epigraphhub_py/blob/6f76dc302f7191e6c86c78196cc18640d3cf5b32/epigraphhub/settings.py#L1

example of use: 

```
from sqlalchemy import create_engine
from epigraphhub.settings import env

with env.db.credentials[env.db.default_credential] as credential:
    engine_public = create_engine(
        f"postgresql://{credential.username}:"
        f"{credential.password}@{credential.host}:{credential.port}/"
        f"{credential.dbname}"
    )
```

It's just necessary to config the credentials

Reply:
Thank you!